### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/1197.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.4
+
+RUN apk add --update ca-certificates \
+    && rm -rf /var/cache/apk/*
```
Comment (kopiczko) [inline Dockerfile]: https://github.com/giantswarm/cert-operator/blob/master/Dockerfile#L3

File: README.md [hunk 2]
```diff
@@ -1,2 +1,5 @@
+[![CircleCI](https://circleci.com/gh/giantswarm/ingress-operator.svg?&style=shield&circle-token=254b6db5aa5ff6bf87ca97b47080b3b972b561f6)](https://circleci.com/gh/giantswarm/ingress-operator)
+
 # ingress-operator
-The ingress-operator connects host cluster ingress controllers with guest cluster ingress controllers on a Giant Swarm Kubernetes host cluster.
+The ingress-operator connects host cluster ingress controllers with guest
+cluster ingress controllers on a Giant Swarm Kubernetes host cluster.
```
Comment (kopiczko) [inline README.md]: Line wrapping <3

File: Dockerfile (hunk 1 above)
Comment (xh3b4sd) [inline Dockerfile]: What do you want to say? Should I change it to the code used in cert-operator? Why is this better than the other?

Comment (kopiczko) [inline Dockerfile]: I just spread the knowledge, there is a newly added flag. Not sure if this is better.

Comment (JosephSalisbury) [inline Dockerfile]: I prefer the no cache stuff

Comment (xh3b4sd) [inline Dockerfile]: Done.

**Pull request merged**